parsec/backend/postgresql/organization.py

```diff
@@ -61,15 +61,28 @@
 """
 )
 
-
 _q_get_stats = Q(
     f"""
 SELECT
+    (
+        SELECT ARRAY(
+            SELECT (revoked_on, profile)
+            FROM user_
+            WHERE organization = { q_organization_internal_id("$organization_id") }
+        )
+    )_users,
     (
         SELECT COUNT(*)
         FROM user_
-        WHERE user_.organization = { q_organization_internal_id("$organization_id") }
+        WHERE user_.organization = { q_organization_internal_id("$organization_id") } AND
+             user_.profile != '{ UserProfile.OUTSIDER.value }'
     ) users,
+    (
+        SELECT COUNT(*)
+        FROM user_
+        WHERE user_.organization = { q_organization_internal_id("$organization_id") } AND
+             user_.profile = '{ UserProfile.OUTSIDER.value }'
+    ) outsiders,
```

Comment:
We don't need to crunch this `outsiders` and the `users` results column given the `SELECT (revoked_on, profile) ...` should give us all the data to retrieve everything ;-)

edit: `result['users']` and `result['outsiders']` are not used in the `stats` function, so those two results columns can safely be removed